### ISSUE TITLE
Protect against user provided field selector

### ISSUE THREAD

**Issue by pke** [felixfbecker/node-sql-template-strings#123]
Is there plans to support something similar to [slonik](https://github.com/gajus/slonik#protecting-against-unsafe-value-interpolation) field selector protection?

**Comment by absolux** [comment]
I don't think so! Slonik is both, a SQL template string helper as this package,  and a layer on top of the `pg` driver for PostgreSQL databases. The feature you ask for is used for query execution not query construction.
